src/components/duration/time.rs

```diff
@@ -418,11 +397,17 @@ impl TimeDuration {
     pub fn iter(&self) -> TimeIter<'_> {
         <&Self as IntoIterator>::into_iter(self)
     }
+
+    /// Returns this `TimeDuration` as a `NormalizedTimeDuration`.
+    pub(crate) fn as_norm(&self) -> NormalizedTimeDuration {
```

Comment:
```suggestion
    pub(crate) fn to_normalized(&self) -> NormalizedTimeDuration {
```

`norm` has a different meaning in math contexts, and the `as_` prefix is commonly used to do simple casts.